nym-api/src/coconut/tests/fixtures.rs

```diff
@@ -48,7 +43,7 @@ pub fn dealer_fixture(mut rng: &mut ChaCha20Rng, id: NodeIndex) -> DealerDetails
     rng.fill_bytes(&mut secondary_seed);
```

Comment:
you can remove the above lines btw since we don't need the secondary rng now

Reply:
Removed in https://github.com/nymtech/nym/pull/4545/commits/dc5a640ad596a3c1d39bd61fa63f3cffdf02a436